FindDistance/src/ua/goit/alg/TestFindDistance.java

```diff
@@ -0,0 +1,32 @@
+package ua.goit.alg;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class TestFindDistance {
+
+  @Test
+  public void twoMinValues() {
+	int[] input = {1,5,0,7,5,90,3,4,0};
+	int expectedDistance = 2;
+	int actualDistance = FindDistance.minDistance(input);
+	assertEquals(expectedDistance, actualDistance);
+  }
+  
+  @Test
+  public void valuesOnStartAndEnd() {
+	int[] input = {1,5,7,5,90,3,4,0};
+	int expectedDistance = 7;
+	int actualDistance = FindDistance.minDistance(input);
+	assertEquals(expectedDistance, actualDistance);
+  }
+  
+  @Test
+  public void theSameValues() {
```

Comment:
Please add a test for an array with a single element and another one for 2 elements